python/ray/serve/controller.py

```diff
@@ -163,10 +163,9 @@ async def shadow_traffic(self, endpoint_name: str, backend_tag: BackendTag,
             self.endpoint_state.shadow_traffic(endpoint_name, backend_tag,
                                                proportion)
 
-    # TODO(architkulkarni): add Optional for route after cloudpickle upgrade
-    async def create_endpoint(self, endpoint: str,
-                              traffic_dict: Dict[str, float], route,
-                              methods: List[str]) -> None:
+    async def create_endpoint(
+            self, endpoint: str, traffic_dict: Dict[str, float],
+            route: Optional[str], methods: List[str]) -> None:
```

Comment:
```suggestion
            route: Optional[str], methods: List[str],) -> None:
```
add a trailing comma and run the formatter again, the format should be slightly better.